### PR TITLE
Feature/application form functionality ln

### PR DESCRIPTION
**Pull request by anewb87**
### 1. What changed?
We now have functionality on our ApplicationFormModal and it behaves like a modal rather than its own page.
Images are brought in from the data.

### 2. Why is this change necessary?
This is part of our MVP. We need the ability to submit an application to our backend. (Our first mutation)

### 3. What changed technically that may impact others?
We connected the ApplicationFormModal to display inside the PetInformation Component.
We created a new application for a pet when a form is submitted.

### 4. How do we test it?
Two new fixtures were created for testing. Stubbed our mutation and the queries when testing if application successfully posted. All functionality of modal has been tested.

### CODE REVIEW

File: cypress/fixtures/applicationSubmission.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        "id": "21",
+        "name": "Loki",
+        "email": "loki66@gmail.com",
+        "description": "I am a super cool guy and my brother is the best. We would take great care of Clifford.",
+        "petId": 2
+      },
+      "errors": []
+    }
+  }
+}
```
Comment (Fordo29) [inline cypress/fixtures/applicationSubmission.json:14]: Great fixture - copied from the BE!  :-)

File: cypress/integration/singlePetPage_spec.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import { getDefaultNormalizer } from "@testing-library/react";
```
Comment (Fordo29) [inline cypress/integration/singlePetPage_spec.js:1]: Do we need this?  Item for review...

Comment (ectweitmann) [inline cypress/integration/singlePetPage_spec.js:1]: Is this being utilized? If not, let's delete it.

File: cypress/integration/singlePetPage_spec.js [hunk 3]
```diff
@@ -6,18 +8,34 @@ describe('Pet Applications Component', () => {
...
+    ).as('updatedApplications');
+
+    cy.intercept(
+      {
+        method: 'POST',
+        url: 'https://not-fur-gotten-be.herokuapp.com/graphql',
+        headers: {
+          'x-gql-operation-name': 'getPetById',
+        },
+        times: 1
```
Comment (Fordo29) [inline cypress/integration/singlePetPage_spec.js:25]: This is such a great learning on how to intercept multiple times.

**Review by ectweitmann — APPROVED** [review]
This is great. Felt like I learned a ton over the course of working on this branch!! Nice work team.

File: src/Components/ApplicationFormModal.js [hunk 4]
```diff
@@ -1,17 +1,58 @@
 import React, {useState} from 'react';
+import { gql, useMutation  } from '@apollo/client';
 import '../Styles/ApplicationFormModal.scss';
 
-const ApplicationFormModal = () => {
+const CREATE_APPLICATION = gql`
```
Comment (Fordo29) [inline src/Components/ApplicationFormModal.js:5]: mutations for GraphQL!  Yaahooooo!  Well done!

**Review by Fordo29 — APPROVED** [review]

**Pull request merged**